src/main.js

```diff
@@ -0,0 +1,32 @@
+var server = require('http').createServer();
```

Comment:
Suggest changing these to not use require:

```
import  http from 'http';
import socketio from 'socket.io';

const server = http.createServer();
const io = socketio(server);
```

Just so it's consistent with everything else.